os_windows.go

```diff
@@ -0,0 +1,28 @@
+// Copyright 2013 The Go Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// +build windows
+
+package ctxvfs
+
+import (
+	"errors"
+	"os"
+	"path/filepath"
+	"strings"
+)
+
+var errInvalidPath = errors.New("Cannot resolve absolute path")
+
+func (root osFS) resolve(path string) (string, error) {
+	// Clean the path so that it cannot possibly begin with ../.
+	// If it did, the result of filepath.Join would be outside the
+	// tree rooted at root.  We probably won't ever see a path
+	// with .. in it, but be safe anyway.
+	path = strings.TrimPrefix(filepath.Clean(filepath.Join(string(root), path)), string(os.PathSeparator))
```

Comment:
this code allows you to escape `root`. Say root is `C:/foo` and path is `../bar`. When this line run path will be `c:/bar`.